docs/src/content/docs/fr/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,295 @@
+
+    ```sh
+    pnpm astro add tailwind
+    ```
+
+    </TabItem>
+
+    <TabItem label="Yarn">
+
+    ```sh
+    yarn astro add tailwind
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+2.  Installez le module d'extension Tailwind de Starlight :
+
+    <Tabs>
+
+    <TabItem label="npm">
+
+    ```sh
+    npm install @astrojs/starlight-tailwind
+    ```
+
+    </TabItem>
+
+    <TabItem label="pnpm">
+
+    ```sh
+    pnpm install @astrojs/starlight-tailwind
+    ```
+
+    </TabItem>
+
+    <TabItem label="Yarn">
+
+    ```sh
+    yarn add @astrojs/starlight-tailwind
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+3.  Créez un fichier CSS pour les styles de base de Tailwind, par exemple dans `src/tailwind.css` :
+
+    ```css
+    /* src/tailwind.css */
+    @tailwind base;
+    @tailwind components;
+    @tailwind utilities;
+    ```
+
+4.  Mettez à jour votre fichier de configuration Astro pour utiliser vos styles de base de Tailwind et désactiver les styles de base par défaut :
+
+    ```js {11-12,16-17}
+    // astro.config.mjs
+    import { defineConfig } from 'astro/config';
+    import starlight from '@astrojs/starlight';
+    import tailwind from '@astrojs/tailwind';
+
+    export default defineConfig({
+    	integrations: [
+    		starlight({
+    			title: 'Documentation avec Tailwind',
+    			customCss: [
+    				// Chemin vers vos style de base de Tailwind:
+    				'./src/tailwind.css',
+    			],
+    		}),
+    		tailwind({
+    			// Désactive les styles de base par défaut:
+    			applyBaseStyles: false,
+    		}),
+    	],
+    });
+    ```
+
+5.  Ajoutez le module d'extension Tailwind de Starlight au fichier `tailwind.config.cjs` :
+
+    ```js ins={2,7}
+    // tailwind.config.cjs
+    const starlightPlugin = require('@astrojs/starlight-tailwind');
+
+    /** @type {import('tailwindcss').Config} */
+    module.exports = {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Mettre en forme Starlight avec Tailwind
+
+Starlight utilise les valeurs de votre [configuration de thème Tailwind](https://tailwindcss.com/docs/theme) dans son interface utilisateur.
+
+Si définies, les options suivantes remplaceront les styles par défaut de Starlight :
+
+- `colors.accent` — utilisé pour les liens et la mise en évidence de l'élément courant
+- `colors.gray` — utilisé pour les couleurs d'arrière-plan et les bordures
+- `fontFamily.sans` — utilisé pour le texte de l'interface utilisateur et du contenu
+- `fontFamily.mono` — utilisé pour les exemples de code
+
+```js {13,16,21,24}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Votre couleur d'accentuation préférée.
```

Comment:
Some references for the translation of "accent"

- https://developer.mozilla.org/fr/docs/Glossary/Accent
- https://learn.microsoft.com/fr-fr/windows/apps/design/signature-experiences/color#accent-color